modules/exploits/windows/http/sharepoint_ssi_viewstate.rb

```diff
@@ -121,7 +126,10 @@ def vuln_builds
   def check
     res = send_request_cgi(
       'method' => 'GET',
-      'uri' => normalize_uri(target_uri.path)
+      'uri' => normalize_uri(target_uri.path),
+      'headers' => {
+        'Cookie' => cookie
+      },
```

Comment:
The API provides something for this already. :)

```suggestion
      'cookie' => cookie
```

Trailing comma can be removed, too.